### ISSUE TITLE
Hundreds of "undefined's"

### ISSUE THREAD

**Issue by orlagwon** [akumidv/tradingview-assistant-chrome-extension#165]
May get 20-60 "undefined" iterations in a row. Sometimes it's just getting stucked in eternal "undefined". Why it happening?

**Comment by VincenzoUrso** [comment]
can happen with some setup that tv is not able to calculate in time

**Comment by scheerd** [comment]
Yeah, this is usually a time-out issue, calculation time too long.  You'll probably see this occasionally when doing manual settings manipulation.  Turn off any extra features or plotting or whatever then try again.

**Comment by akumidv** [comment]
@orlagwon you can increase timeout in options
![image](https://github.com/akumidv/tradingview-assistant-chrome-extension/assets/12490281/829a54b2-65e1-4a02-8aa7-3dc1fc3961fa)

**Comment by orlagwon** [comment]
@akumidv TradingView calculates normally, I see data in strategy tester window, but assistant says "undefined".

**Comment by xuyamu** [comment]
> @akumidv TradingView calculates normally, I see data in strategy tester window, but assistant says "undefined".

I may have encountered this problem. This problem occurs when performing Deep Backtesting. I don’t know if it is really performing Deep Backtesting.

**Comment by akumidv** [comment]
Deep backtesing was fixes in 2.10.x version, in last 2.11.x no one are mention this bug.
